### DISCUSSION TITLE
Does the plugin actively engages in data collection while running as a background service?

### DISCUSSION THREAD

**Discussion by steffomix** [builttoroam/device_calendar#521] [Q&A]
Hello, I am using this plugin since over a year for my private purposes and want to publish my app to Google Play. But I am curious about in how much it can be named as data collection if the plugin writes user data and location data automatically in background to the device calendar. I am looking this up since a while but I can nowhere find an example or something alike to get at least a hint with what I am dealing in this topic.

So perhaps someone from the community can give a little help or hint on that.
Thank you!

**Comment by thomassth** [comment]
This plugin by itself does not collect anything. 

Most people would use it with a calendar provider however, like Google calendar, and they would have their own privacy policies.

**Reply by steffomix** [reply]
Google play says that it is data collecting even if your app sends data to another app on the same device. 
However, I use this plugin for me on my self made app and I am very happy with it. But I was not able to get together a privacy policy that covers this issue without spending hundreds of dollars for a lawyer.
So I wonder if someone has some experiences with this or know an app on google play that uses this plugin, so that others like me can learn from it how to handle the privacy policy issues.
